web/main.py

```diff
@@ -100,7 +100,7 @@ def handler(dataset=None, filename=None):
     if blob is None:
         return abort(404, 'File was not found')
 
-    response = Response(blob.download_as_string())
+    response = Response(stream_with_context(blob.download_as_text()))
```

Comment:
I guess not, but not way we can stream without loading the whole file into memory first?

Reply:
Yeah, I thought the same, but wasn't sure whether a simple `open()` will work as expected. I'll give it a try.

Reply:
Seems to be working 👍